server/src/main/java/org/opensearch/tracing/OSSpan.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.tracing;
+
+import io.opentelemetry.api.trace.SpanContext;
+
+/**
+ * Default implementation of {@link Span}. It keeps a reference of OpenTelemetry Span and handles span
+ * lifecycle management by delegating calls to it.
+ */
+class OSSpan implements Span {
```

Comment:
should this be DefaultSpan following the same terminology for DefaultTracer?

Reply:
Done